CONTRIBUTING.md

```diff
@@ -16,12 +16,20 @@ Finally, run tox with `tox`.  All tests should pass!
 
 To add a new dataset loader:
 
-1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is!)
-2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical"!)
+1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is)
+2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical") 
 3. Create a module in mirdata, e.g. `mirdata/my_dataset.py`
 4. Create tests for your loader in `tests/`, e.g. `test_my_dataset.py`
 5. Add your module to `docs/source/mirdata.rst`
 6. Add the module to `mirdata/__init__.py`
+7. Add the module to the table in the `README.md` file, section `Currently supported datasets`
+8. Make sure the dataset appears in `https://github.com/ismir/mir-datasets`
```

Comment:
unclear what "make sure" means. What do i do if the dataset doesn't appear there?

Reply:
True, alse decided with @rabitt to remove this step for now in the template, so removing it from here as well